LocalMultiplayerAgent/Controllers/SessionHostController.cs

```diff
@@ -96,6 +96,29 @@ public SessionHostController(IHostApplicationLifetime appLifetime)
             });
         }
 
+        private static bool _wasGsdkVersionLogged = false;
+        [HttpPost]
+        [Route("v1/metrics/{sessionHostId}/gsdkinfo")]
+        public IActionResult ReportGsdkVersion(string sessionHostId, [FromBody] GsdkVersionInfo vi)
+        {
+            if (string.IsNullOrEmpty(vi.Version))
+            {
+                return BadRequest($"{nameof(GsdkVersionInfo.Version)} should not be an empty string");
+            }
+            if (string.IsNullOrEmpty(vi.Flavor))
+            {
+                return BadRequest($"{nameof(GsdkVersionInfo.Flavor)} should not be an empty string");
+            }
+
+            if (!_wasGsdkVersionLogged)
+            {
+                Globals.MultiLogger.LogInformation($"GSDK flavor/version: {vi.Flavor}/{vi.Version} from session host {sessionHostId}");
```

Comment:
I think we just do Console.WriteLine on LMA, does this work?

Reply:
It does. The multilogger is setup with a console sink.